src/hotspot/share/gc/shared/taskqueue.hpp

```diff
@@ -116,7 +116,7 @@ void TaskQueueStats::reset() {
 
 // TaskQueueSuper collects functionality common to all GenericTaskQueue instances.
 
-template <unsigned int N, MEMFLAGS F>
+template <unsigned int N, MemTag F>
```

Comment:
MemTag parameter name should probably be changed here and elsewhere in taskqueue code.
Suggest `mem_tag`.

Reply:
I was going to suggest just MT which is more in keeping with the short/terse names given to type parameters.

Reply:
Will be address in a followup issue.